SU-SWS/decanter
# Add new colors from identity guide and deprecate old color

# READY FOR REVIEW

# Summary
- Adding new identity Guide colors to v6 https://identity.stanford.edu/design-elements/color/
- Deprecate colors that will be removed (no longer in Identity Guide)
- There are some colors that get to stay but have new names (added deprecation message)
- Update our color maps (core, nav, alerts) to use new color names and make sure alert component pulls colors from color map.
- Following proper sassdoc convention, remove $ sign in the `@name` field for the variable, add `@type Color` to all color variables.
Note: I have tested this branch for the Redwood WP repo and the colors look normal to me.
Note2: The variable for color Fog (and its variants) is called `$su-color-foggy` since we had `$su-color-fog` (which has become `$su-color-foggy-light `in v7) in v6 which was used very frequently. We discussed this in a Decanter meeting and I feel like this option could avoid some confusion in code.

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Look at the files changed
1. Make sure to expand (click "Load diff" under the file, in the GitHub Files changed tab) to see below file, since that's where most of the new code is:
`core/src/scss/utilities/variables/core/_colors.scss`
1. Also check the color map scss files and see that they're using the new color variables properly
1. You can also test this branch on one of your projects that uses Decanter by setting the branch to 
git://github.com/SU-SWS/decanter.git#611-new-colors

# Affected Projects or Products
- many things that uses decanter - d8, Redwood etc

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
